autocannon.js

```diff
@@ -71,7 +74,11 @@ function start () {
   })
 
   if (!argv.json) {
-    track(tracker)
+    var render = {
+      renderProgressBar: argv.progress,
+      renderLatencyTable: argv.latency
+    }
+    track(tracker, render)
```

Comment:
You can define this option object inline.
In fact, I would alias `--latency` to `renderLatencytable` and `--progress` to `renderProgressBar`, and pass argv directly.